tools/README.md

```diff
@@ -8,7 +8,11 @@
 
 ## Vim plugins
 
-- ...
+- [syntastic](https://github.com/scrooloose/syntastic) - syntax checking plugin
```

Comment:
one downside lang to syntastic: ruby is on by default, ruby is slow, therefore vim editing ruby gets slow (unless you turn it off yourself). though haven't used it in a while, so i'm not sure if this is still the case :)